pandas/core/arrays/timedeltas.py

```diff
@@ -288,6 +317,21 @@ def _evaluate_with_timedelta_like(self, other, op):
 
         return NotImplemented
 
+    __mul__ = _wrap_tdi_op(operator.mul)
+    __rmul__ = __mul__
+    __truediv__ = _wrap_tdi_op(operator.truediv)
+    __floordiv__ = _wrap_tdi_op(operator.floordiv)
+    __rfloordiv__ = _wrap_tdi_op(ops.rfloordiv)
```

Comment:
how are you implmenting rfloordiv but not rdiv / rtruediv?

Reply:
very specifically implementing only the methods needed to extend the affected tests.  I'll double-check why i couldn't extend tests for rdiv and rtruediv

Reply:
AFAICT we don't have test cases for the other methods, at least not in tests.arithmetic.test_timedelta64.  I've scoured the other test files to centralize arithmetic tests, but it's conceivable I've missed some.

We definitely need such tests, but I'd rather do that in a dedicated PR, and would prefer not to implement the methods here without the corresponding tests.

Reply:
> We definitely need such tests, but I'd rather do that in a dedicated PR, and would prefer not to implement the methods here without the corresponding tests.

+1